src/br/com/levelupacademy/models/subcategory/Subcategory.java

```diff
@@ -47,13 +47,17 @@ public int getSequence() {
         return sequence;
     }
 
-    public Category getCategory() {
-        return category;
-    }
-
     public String getCategoryCode() {
         return category.getCode();
     }
+
+    public boolean hasDescription(String description) {
+        if(description.isEmpty() || description.isBlank() || description.equals("")) {
+            return false;
+        } else {
+            return true;
+        }
+    }
```

Comment:
Aqui você não precisa receber a descrição para verificá-la, pois ela já é um atributo da sua própria classe. Bastaria você fazer essa verificação diretamente:
```suggestion
    public boolean hasDescription() {
        return !description.isEmpty() && !description.isBlank();           
    }
```

Reply:
nesse caso seria && ou || pois basta que uma delas atenda ne ?

Reply:
Acho que podemos usar só o `isBlank` e verificar também se não é nulo: `return description != null && !description.isBlank()`. O que acha?

Reply:
Sim fica uma verificação legal